ccf_transaction_processor/Readme.md

```diff
@@ -6,37 +6,142 @@ https://creativecommons.org/licenses/by/4.0/
-the "get_ledger_verifying_key" transaction. This feature may be used by PDO clients to establish offline verifiable proof of transaction commits as desired by the PDO smart contract application. Note that for trust purposes, it is recommended that any entity that uses the above verifying_key gets it directly from the CCF service using the
-"get_ledger_verifying_key" transaction.
-
-We note that the very first invocation of "get_ledger_verifying_key" rpc
-after the service is started is used to create the key pair, which will then be globally committed.
-The system admin who deploys the CCF ledger is expected to invoke the "get_ledger_verifying_key"
-rpc after deployment to generate these keys. Further, the admin must ensure that this key pair is globally committed
-before making the ledger available for PDO usage. This can checked by issuing the rpc again & ensuring that a verifying key gets returned. See https://microsoft.github.io/CCF/users/issue_commands.html for instructions on
-how an rpc can be issued via command line.
+The software is located under $PDO_SRC/transaction_processor/. The TP software is written and tested for CCF tag 0.9.2. Compatability with other CCF versions is not guaranteed.
+
+The instructions below may be followed to build and deploy CCF based PDO TP.
+
+# Get CCF Source Code
+
+CCF tag 0.9.2 is included as a submodule within PDO. Download the submodule using the following commands:
+
+```bash
+cd $PDO_SRC
+git submodule init
+git submodule update
+```
+
+# Install CCF Dependencies
+
+CCF/PDO combo has been tested under a scenario where CCF is deployed in a standalone VM. Further, the CCF/PDO combo
+has been tested only for the CCF virtual enclave mode. The dependencies needed to deploy CCF in an Ubuntu 18.04  VM
+with  virtual enclave mode can be installed by running the following command:
+
+```bash
+cd $PDO_SRC/ccf_transaction_processor/CCF/getting_started/setup_vm/
+./setup_nosgx.sh
+```
+
+The above script works only when the VM is not behind any proxies. If there are proxies, make the following changes
+before executing the above comamnd:
+
+	A. In the setup_nosgx.sh file, modify the command `sudo add-apt-repository ppa:ansible/ansible -y` by adding the `-E` option.
+           The modified command looks like `sudo -E add-apt-repository ppa:ansible/ansible -y`. Add a `sudo`
+           prefix to the `ansible-playbook ccf-dependencies-no-driver.yml` command appearing in the same file.
+
+	B. In ccf-dependencies-no-driver.yml (present under the same folder), add the environment option for proxies
+           (see https://docs.ansible.com/ansible/latest/user_guide/playbooks_environment.html for reference)
+
+                environment:
+                    http_proxy: <specify-http-proxy-here>
+                    https_proxy: <specify-https-proxy-here>
+
+	C. Add the repo used by ansible scripts for installing python 3.7
+
+		```bash
+		sudo add-apt-repository ppa:deadsnakes/ppa
+		```
```

Comment:
Something is still wrong in the markup, at least as far as seen in https://github.com/hyperledger-labs/private-data-objects/blob/46fc778cbe3225b2b727315fb060b602667f4e8a/ccf_transaction_processor/Readme.md:  the whole list already seems to be in some quote/verbatim mode and then is is shown literally including the back-quotes ...

Reply:
thank you, removed all indents.

Reply:
apparently, it is  possible to preview .md files in vscode, never knew this until now!